source/reference/versioning.txt

```diff
@@ -82,6 +82,69 @@ released, version 2.0 would no longer receive patch releases. At the
 same time, version 1.3 would continue to receive patches until version
 1.4 was released.
 
+.. _c2c-versioning-live-upgrade:
+
+Live Upgrade
+~~~~~~~~~~~~
+
+.. versionadded:: 1.6.0
+
+Starting in ``mongosync`` 1.6.0, you can upgrade ``mongosync`` without
+restarting data synchronization operations from the beginning. This is
+known as a *live upgrade*. After the live upgrade, ``mongosync``
+continues operations that were in progress before the upgrade.
+
+You can live upgrade ``mongosync`` to the latest version.
+
+To live upgrade ``mongosync``, perform these steps:
+
+.. procedure::
+   :style: normal
+
+   .. step:: (Optional) Pause the ``mongosync`` processes
+   
+      To pause ``mongosync``, run :ref:`c2c-api-pause` on each running
+      ``mongosync`` process.
+      
+      You can restart the ``mongosync`` processes later from the paused
+      state, as shown in the final step.
+      
+      .. tip::
+
+         If you don't want to immediately resume data synchronization
+         operations after live upgrade, consider pausing the
+         ``mongosync`` processes. This is useful if you're coordinating
+         operations from multiple ``mongosync`` processes.
+
+   .. step:: Stop the ``mongosync`` processes
+   
+      Use your operating system to stop each running ``mongosync``
+      process. To stop a process, see your operating system
+      documentation for the specific steps.
+
+   .. step:: Download and install the new ``mongosync`` binaries
+   
+      To download and install the new ``mongosync`` binaries, see
+      :ref:`download and install mongosync <c2c-install>`.
+
+   .. step:: Initialize the ``mongosync`` processes
+
+      To initialize the ``mongosync`` processes, see the
+      :ref:`mongosync examples <c2c-mongosync-examples>`.
+      During initialization, ``mongosync`` updates the persistent
+      metadata.
+
+   .. step:: Resume the ``mongosync`` processes
+
+      If you paused any ``mongosync`` processes earlier in the optional
+      first step, run :ref:`c2c-api-resume` on all ``mongosync``
+      processes.
+
+If a data synchronization operation was already in progress before the
+live upgrade, new features added in the latest ``mongosync`` version
+may not all be supported after the live upgrade. Performance
+improvements and bug fixes are always supported after the live upgrade.
+
 Upgrade and Downgrade
```

Comment:
This header should only say `Downgrade` since we now support `Live Upgrade` (which is described in the section above)